README.md

```diff
@@ -0,0 +1,4 @@
+# Mobile Services Installer
+
+Prerequisites:
+* Ansible 2.7.6
```

Comment:
we should add a bit more info here about how to use the installer, both locally or against an existing cluster.

Reply:
agree